ibis/pandas/dispatch.py

```diff
@@ -42,8 +42,9 @@ def execute_node_without_scope(node, **kwargs):
 
 
 # Default returns an empty scope
+@pre_execute.register(ops.Node)
 @pre_execute.register(ops.Node, ibis.client.Client)
-def pre_execute_default(node, client, **kwargs):
+def pre_execute_default(node, *clients, **kwargs):
```

Comment:
Is this the new variadic dispatch? Or just the first `client` needs to be an instance of `ibis.client.Client`?

Reply:
No, not variadic. `ibis.pandas.execute(ibis.now())` has no client, and yet it still needs to be able to `pre_execute`.

Reply:
Ahh, I see now.